config/locales/views/transactions/fr.yml

```
@@ -26,47 +31,178 @@ fr:
+      pending: En attente
+      pending_tooltip: Transaction en attente — susceptible de changer une fois enregistrée
+      linked_with_provider: Lié à %{provider}
+      activity_type_tooltip: Type d'opération d'investissement
+      possible_duplicate: Doublon ?
+      potential_duplicate_tooltip: Il s'agit peut-être d'un doublon d'une autre transaction
+      review_recommended: À vérifier
+      review_recommended_tooltip: Écart de montant important — vérifiez s'il s'agit d'un doublon
+      split: Divisée
+      split_tooltip: Cette transaction a été divisée en plusieurs entrées
+      split_child_tooltip: Fait partie d'une transaction divisée
+    merge_duplicate:
```

YAMLlint (1.38.0)

[error] duplication of key "merge_duplicate" in mapping

(key-duplicates)